Go/Go.sublime-syntax

```diff
@@ -774,11 +783,13 @@ contexts:
 
   match-keyword-continue:
     - match: \bcontinue\b
-      scope: keyword.control.go
+      scope: keyword.control.flow.break.go
 
   match-keyword-default:
-    - match: \bdefault\b
-      scope: keyword.control.go
+    - match: \b(default)\b(:)?
+      captures:
+        1: keyword.control.switch.case.go
```

Comment:
```suggestion
        1: keyword.control.conditional.default.go
```

I guess we could argue about using 

1. `keyword.control.conditional.case` 
2. `keyword.control.conditional.switch.case`
3. `keyword.control.conditional.switch.default`

as alternative scopes, but other syntaxes just use the plain `default`. IIRC that was because of different possible statement structures, some of them not using `switch` keyword.